components/camel-geocoder/src/test/java/org/apache/camel/component/geocoder/GeoCoderComponentAddressTest.java

```diff
@@ -39,10 +39,10 @@ protected RouteBuilder createRouteBuilder() throws Exception {
         return new RouteBuilder() {
             public void configure() {
                 from("direct:start")
-                    .to("geocoder:address:Paris, France")
-                    .to("log:result")
-                    .log("Location ${header.CamelGeocoderAddress} is at lat/lng: ${header.CamelGeocoderLatlng} in city ${header.CamelGeocoderCity}")
-                    .to("mock:result");
+                        .to("geocoder:address: empty?apiKey=" + getApiKey())
```

Comment:
tests do no run. not sure about this uri setting.

Reply:
@onderson are there other tests that require secrets?

Reply:
@jborza , not sure, as i said no tests were run while doing the build for the component.